assets/js/aplus.js

```diff
@@ -13,6 +13,10 @@
  window.CustomEvent = CustomEvent;
 })();
 
+let currentScript =  (document.currentScript ?
```

Comment:
This shouldn't be in the global namespace. Move it inside the local namespace.

Reply:
Actually, maybe not. We can move the translations to use JS api, so we don't need to pass arguments to single component.